packages/mobile/src/harmony-native/components/Skeleton.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from 'react'
+
+import { useTheme } from '@emotion/react'
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+  Easing
+} from 'react-native-reanimated'
+
+import { Box } from './layout/Box/Box'
+import type { BoxProps } from './layout/Box/types'
+
+const AnimatedBox = Animated.createAnimatedComponent(Box)
+
+type SkeletonProps = BoxProps
+
+export const Skeleton = (props: SkeletonProps) => {
+  const { color } = useTheme()
+  const color1 = color.neutral.n50
+  const color2 = color.neutral.n100
+
+  // Create shared value for shimmer animation
+  const shimmerPosition = useSharedValue(0)
+
+  // Setup animation loop
+  useEffect(() => {
+    shimmerPosition.value = withRepeat(
+      withSequence(
+        withTiming(1, {
+          duration: 1500,
+          easing: Easing.linear
+        }),
+        withTiming(0, {
+          duration: 0
+        })
+      ),
+      -1 // Infinite repeat
+    )
+  }, [shimmerPosition])
+
+  // Create animated styles for the shimmer effect
+  const shimmerStyle = useAnimatedStyle(() => {
```

Comment:
do we want a `disableShimmer` option for mobile? I know it was a perf concern sometimes

Reply:
yes! adding